### PR TITLE
document CAP_SYS_ADMIN required for systemd PrivateNetwork

### PR DESCRIPTION
**Pull request by jamescassell**
This wasn't working for rootful containers either without the `--privileged` flag.  I did some searching and found that `CAP_SYS_ADMIN` is required for `PrivateNetwork` feature to work: https://stackoverflow.com/questions/39539188/why-unshareclone-newnet-requires-cap-sys-admin

### CODE REVIEW

**Comment by jamescassell** [comment]
original "PrivateNetwork doesn't work in rootless" reference in the docs was added in https://github.com/containers/libpod/pull/3972

**Comment by rhatdan** [comment]
Thanks @jamescassell, but you need to sign your PR.
git commit -a --amend -s
git push --force
LGTM

File: rootless.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ can easily fail
   * As of Fedora 31 defaults to cgroup V2, which has full support of rootless cgroup management.  Note this requires the --cgroup-manager within rootless containers to use systemd, which new containers will get by default.
 * Some system unit configuration options do not work in the rootless container
   * systemd fails to apply several options and failures are silently ignored (e.g. CPUShares, MemoryLimit). Should work on cgroup V2.
-  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).
+  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).  systemd services requiring `PrivateNetwork` can be made to work by passing `--cap-add SYS_ADMIN`, but the security implications should be carefully evaluated. In most cases, it's better to create an override.conf dropin that sets `PrivateNetwork=no`  This applies also for root containers.
```
Comment (TomSweeneyRedHat) [inline rootless.md:20]:
```suggestion
  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).  The systemd services requiring `PrivateNetwork` can be made to work by passing `--cap-add SYS_ADMIN`, but the security implications should be carefully evaluated. In most cases, it's better to create an override.conf dropin that sets `PrivateNetwork=no`.  This also applies to containers run by root.
```

**Comment by vrothberg** [comment]
The tests are currently failing because an container image we use in the CI has changed and regressed. Can you rebase the PR on the lastet master? This will unblock CI.

**Comment by jamescassell** [comment]
Seems like CI should be testing a merge, not the PR HEAD....

**Comment by rhatdan** [comment]
@jamescassell Please rebase.

File: rootless.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ can easily fail
   * As of Fedora 31 defaults to cgroup V2, which has full support of rootless cgroup management.  Note this requires the --cgroup-manager within rootless containers to use systemd, which new containers will get by default.
 * Some system unit configuration options do not work in the rootless container
   * systemd fails to apply several options and failures are silently ignored (e.g. CPUShares, MemoryLimit). Should work on cgroup V2.
-  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).
+  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).  The systemd services requiring `PrivateNetwork` can be made to work by passing `--cap-add SYS_ADMIN`, but the security implications should be carefully evaluated. In most cases, it's better to create an override.conf drop-in that sets `PrivateNetwork=no`.  This also applies to containers run by root.
```
Comment (QiWang19) [inline rootless.md:20]:
```suggestion
  * Use of certain options will cause service startup failures (e.g. PrivateNetwork).  The systemd services requiring `PrivateNetwork` can be made to work by passing `--cap-add SYS_ADMIN`, but the security implications should be carefully evaluated.  In most cases, it's better to create an override.conf drop-in that sets `PrivateNetwork=no`.  This also applies to containers run by root.
```

**Comment by TomSweeneyRedHat** [comment]
@jamescassell still missing a signature on your commit `git commit --amend -s` and the gating tests won't let it through until it gets one.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM otherwise.

**Comment by vrothberg** [comment]
@jamescassell can you squash the two commits and repush? That should unblock CI and turn it green.

**Pull request closed without merging**